js/controllers/wizard/WizardHandler.js

```diff
@@ -586,5 +585,27 @@
 	};
 
 
+	/**
+	 * Disable the continue button
+	 *
+	 * @private
```

Comment:
I'm not sure why the linter didn't flag this, but with the `@private` flag, the function name will need a `_` suffix.